lib/src/sql/statements/show.rs

```diff
@@ -17,6 +18,19 @@ pub enum ShowSince {
 	Versionstamp(u64),
 }
 
+impl ShowSince {
```

Comment:
Having some test for ShowSince would be nice 👍 
Probably converting to/from versionstamp using both methods in a single test would be enough?

Reply:
Added test for vs<>u64 as well as datetime => None as timestamp